packages/next/build/index.ts

```diff
@@ -16,7 +17,10 @@ function collectPages(
   directory: string,
   pageExtensions: string[]
 ): Promise<string[]> {
-  return recursiveReadDir(directory, new RegExp(`\\.(?:${pageExtensions.join('|')})$`))
+  return recursiveReadDir(
+    directory,
+    new RegExp(`\\.(?:${pageExtensions.join('|')})$`)
+  )
```

Comment:
Why do we keep getting diffs like this

Reply:
Because we don't have prettier turned on for our codebase.